CHANGELOG.md

```diff
@@ -14,7 +14,62 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Breaking changes
 
 * Removed `join_alias`
-* Change `in_tuples` interface to accept `IntoValueTuple` https://github.com/SeaQL/sea-query/pull/386
+* Changed `in_tuples` interface to accept `IntoValueTuple` https://github.com/SeaQL/sea-query/pull/386
+* **Changed `cond_where` semantics**
+```rust
+// Before: will extend current Condition
+assert_eq!(
+    Query::select()
```

Comment:
@tyt2y3 can you rewrite examples using macros: `all!` and `any!`? 

I think it's easier to read)

Reply:
changed